vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -154,3 +154,320 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
         [![CustomNavigation]][CustomNavigation]
 
     [CustomNavigation]: ../../assets/user_guides/navigation/custom_navigation.png
+
```

Comment:
Not your code, but should we remove

```py
Dashboard(
    pages=[page_1, page_2],
    navigation=Navigation(pages={
        "First title": ["My first page"],
        "Second title": ["My second page"]})
)
```
above, it confused me more than helping, because its almost the same as the code block above.

We could just write: " Insert this Navigation model ..."

Reply:
Also, shall we just delete the first example? It doesn't really provide anything new, but makes the page big